### PR TITLE
RD-3980: Updated translation labels

### PR DESCRIPTION
**Pull request by NorbertAmartus**
## Description

Changes consists of modifying a few labels, simple as that 😁 

## Screenshots / Videos

I'm attaching the screenshots after the changes, to confirm, that all of the places (pointed out in the ticket) has been addressed
Create blueprint from Terraform
Upload from Terraform module
CPT2201180908-1814x286

## Checklist

- [x] My code follows the UI Code Style.
- [ ] I verified that all tests and checks have passed.
- [ ] I verified if that change requires a documentation update.
- [x] I added proper labels to this PR.

## Tests

 

As the changes was applied only to the labels (and I've updated the tests, related to those labels, accordingly) I did not execute System Tests

## Documentation

Documentation will be updated while progressing on [this task](https://cloudifysource.atlassian.net/browse/RD-3969)

### CODE REVIEW

**Comment by NorbertAmartus** [comment]
### Just a note

All of the discussion under [this task](https://cloudifysource.atlassian.net/browse/RD-3980) has been resolved.   
Because of that I'm resolving the "draft" status of this PR 🎉

File: templates/pages/blueprints.json [hunk 1]
```diff
@@ -113,7 +113,7 @@
                     "configuration": {
                         "icon": "",
                         "imageSrc": "/console/static/images/terraform_logo.png",
-                        "label": "Terraform templates",
+                        "label": "Terraform Modules",
```
Comment (qooban) [inline templates/pages/blueprints.json:116]: Optional:

It does not matter much (as it is uppercased anyway), but maybe for the sake of order it should be "Terraform modules".

File: templates/pages/adminDash.json [hunk 2]
```diff
@@ -1,149 +1,153 @@
 {
-  "name": "Dashboard",
```
Comment (qooban) [inline templates/pages/adminDash.json:2]: Seems like some whitespaces changes were introduced here. Was it intentional?

**Review by qooban — COMMENTED** [review]
Looks good. Added only minor comments.

I think that, even though it's simple, you can run system tests only for affected spec (it should not take a lot of time to get the results).

File: templates/pages/adminDash.json (hunk 2 above)
Comment (NorbertAmartus) [reply templates/pages/adminDash.json:2]: Nope, my IDE is doing some pranks on me again 😁 
I'll restore the previous whitespaces

File: templates/pages/blueprints.json (hunk 1 above)
Comment (NorbertAmartus) [reply templates/pages/blueprints.json:116]: Sure, keeping things consistent is always good 🚀

File: templates/pages/adminDash.json (hunk 2 above)
Comment (NorbertAmartus) [reply templates/pages/adminDash.json:2]: Whitespaces was restored 🎉

File: templates/pages/blueprints.json (hunk 1 above)
Comment (NorbertAmartus) [reply templates/pages/blueprints.json]: Done 😉

**Review by qooban — APPROVED** [review]
Thanks for addressing all comments 👍🏻

**Pull request merged**